msyncd/AccountsHelper.cpp

```diff
@@ -100,7 +100,6 @@ void AccountsHelper::slotAccountCreated(Accounts::AccountId id)
                 delete syncProfile;
             }
         }
-        delete newAccount;
     }
```

Comment:
We have two options: 1) change the code to call fromId() instead of manager->account() - but this would then REQUIRE us to manually manage the lifetime of the returned account (ie, delete it); 2) leave the code as manager->account() but ensure that we do NOT attempt to delete / deleteLater the account.

I went with option 2, in this case, for simplicity.  It does mean that if there are a large number of accounts on the device, we can have that many account objects in memory for the lifetime of the msyncd process, but it avoids the possibility of either leaks or crashes, and also should result in a (very minor) performance improvement due to less allocations and less signal connections/disconnections.